app/components/admin/edition_images/lead_image_component.html.erb

```diff
@@ -0,0 +1,55 @@
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Lead image",
+  font_size: "m",
+  padding: true,
+} %>
+
+<%= render "govuk_publishing_components/components/details", {
+  title: "Using a lead image",
+} do %>
+  <% lead_image_guidance %>
+<% end %>
+
+<% if lead_image.present? || case_study? %>
+  <div class="govuk-grid-row">
+    <% if lead_image.present? %>
+      <div class="app-c-edition-images-lead-image-component__lead_image">
+        <div class="govuk-grid-column-one-third">
+          <img src="<%= lead_image.url %>" alt="Lead image" class="app-view-edition-resource__preview">
+          <% unless lead_image.image_data&.all_asset_variants_uploaded? %>
+            <span class="govuk-tag govuk-tag--green">Processing</span>
+          <% end %>
+        </div>
+
+        <div class="govuk-grid-column-two-thirds">
+          <p class="govuk-body"><strong>Caption: </strong><%= caption %></p>
+          <p class="govuk-body"><strong>Alt text: </strong><%= alt_text %></p>
+        </div>
+      </div>
+    <% end %>
+
+    <div class="app-view-edition-resource__actions govuk-grid-column-full govuk-button-group">
+      <% if case_study? %>
+        <%= form_with(url: update_image_display_option_admin_edition_path(edition), method: :patch) do |form| %>
```

Comment:
previously this was wrapped in the form so there was a bit less duplication. It feels wrong to render a form that can't be submitted/isn't relevant to the edition type though. Interested to see what people think.

Reply:
Do the links need to be inside the form when it is a case study? It looks like they should always render when the lead image is present, so could they just go outside of the case study condition? Appreciate we might have to tweak some styles